### PR TITLE
updates to file

### PR DESCRIPTION
**Pull request by pjbrown9**
2nd try

### CODE REVIEW

File: pjbrown-beach park.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Beack Park. IL
+Northeast of _Six Flags_
 
 ### Restaurants
-There are many to choose from, along with many types
+Red Roin
+Steak N Shake
 
 ### Attractions
-Six Flags
```
Comment (psaweh) [inline pjbrown-beach park.md:7]: Please add attraction city

**Comment by pjbrown9** [comment]
@psaweh  please load this crazy thing!

**Comment by pjbrown9** [comment]
okay

**Review by psaweh — APPROVED** [review]
cool

**Review by pjbrown9 — COMMENTED** [review]
done

**Pull request merged**